File: PreviousRobotsJava/blocks/2021 Manual Op mode.js
var lifter, push, FL, servo, FR;

/**
 * This function is executed when this Op Mode is selected from the Driver Station.
 */
function runOpMode() {
  linearOpMode.waitForStart();
  if (linearOpMode.opModeIsActive()) {
  }
  while (linearOpMode.opModeIsActive()) {
    if (gamepad2.getDpadUp()) {
      lifter = -1;
    } else if (gamepad2.getDpadDown()) {
      lifter = 1;
    } else {
      lifter = 0;
    }
    if (gamepad2.getX()) {
      push = -1;
    } else if (gamepad2.getY()) {
      push = 1;
    } else {
      push = 0;
    }
    if (gamepad2.getA()) {
      TheClawAsServo.setPosition(0);
      servo = 0;
    } else if (gamepad2.getB()) {
      TheClawAsServo.setPosition(45);
      servo = 45;
    }
    FL = gamepad1.getLeftStickY();
    FR = -gamepad1.getRightStickY();
    FLAsDcMotor.setPower(FL);
    FRAsDcMotor.setPower(FR);
    LifterAsDcMotor.setPower(lifter);
    ExtendAsDcMotor.setPower(push);
    telemetry.addNumericData('Lifter', lifter);
    telemetry.addNumericData('FR', FR);
    telemetry.addNumericData('FL', FL);
    telemetry.addNumericData('Extender', push);
    telemetry.addNumericData('Servo', servo);
    telemetry.update();
  }
}

File: OldJava/src/org/firstinspires/ftc/teamcode/_2021ManualOpmode.java
package org.firstinspires.ftc.teamcode;

import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
import com.qualcomm.robotcore.hardware.DcMotor;

@TeleOp(name = "_2021ManualOpmode (Blocks to Java)")
public class _2021ManualOpmode extends LinearOpMode {

  private DcMotor FL;
  private DcMotor FR;

  /**
   * This function is executed when this Op Mode is selected from the Driver Station.
   */
  @Override
  public void runOpMode() {
    FL = hardwareMap.get(DcMotor.class, "FL");
    FR = hardwareMap.get(DcMotor.class, "FR");

    // Put initialization blocks here.
    waitForStart();
    if (opModeIsActive()) {
      // Put run blocks here.
      while (opModeIsActive()) {
        // Put loop blocks here.
        telemetry.update();
        if (0 < gamepad1.left_stick_y) {
          FL.setPower(1);
        } else {
          FL.setPower(0);
        }
        if (0 > gamepad1.left_stick_y) {
          FL.setPower(-1);
        } else {
          FL.setPower(0);
        }
        if (0 < gamepad1.right_stick_y) {
          FR.setPower(1);
        } else {
          FL.setPower(0);
        }
        if (0 > gamepad1.right_stick_x) {
          FR.setPower(-1);
        } else {
          FL.setPower(0);
        }
      }
    }
  }
}
